src/content/insertable-streams/video-processing/index.html

```diff
@@ -85,18 +85,18 @@ <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC
 </div>
 
 <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
-<script src="js/camera-source.js" defer></script>
-<script src="js/canvas-transform.js" defer></script>
-<script src="js/peer-connection-pipe.js" defer></script>
-<script src="js/peer-connection-sink.js" defer></script>
-<script src="js/peer-connection-source.js" defer></script>
-<script src="js/pipeline.js" defer></script>
-<script src="js/simple-transforms.js" defer></script>
-<script src="js/video-mirror-helper.js" defer></script>
-<script src="js/video-sink.js" defer></script>
-<script src="js/video-source.js" defer></script>
-<script src="js/webgl-transform.js" defer></script>
-<script src="js/main.js" defer></script>
+<script src="js/camera-source.js" async></script>
```

Comment:
Are any of these dependent on order and that the page has completely loaded(looks like everything is now triggered by the onload event thus I assume no but I just want to make sure)?

Reply:
That's right. Now, the only non-declaration in any of the files is:
window.onload = initUI;